PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -301,6 +301,15 @@
       <add-to-group group-id="RunContextPopupGroup" anchor="last"/>
     </action>
 
+    <group id="Actions.BindServicePopupGroup" text="Bind Service" description="Bind Service" icon="/icons/azure.png" popup="true" class="com.microsoft.intellij.actions.BindServicePopupGroup">
```

Comment:
where's the icon file and Action classes?

Reply:
will PR it in later PR.